app/controllers/api/v1/users_controller.rb

```diff
@@ -19,7 +19,7 @@ def create
 
       def show
         # params = { auth: { provider: , uid: } }
-        return error_message unless params[:auth]
+        return render_message unless params[:auth]
 
         @user = User.from_omniauth(params.fetch(:auth))
```

Comment:
use memoization to user or callbacks to create and show actions

Reply:
done